### DISCUSSION TITLE
which exact metadata tags does Voice use?

### DISCUSSION THREAD

**Discussion by chaserene** [PaulWoitaschek/Voice#2456] [Q&A]
I want to tag my audio and video files such that the metadata (mainly the author, title and chapter) is displayed in Voice.

which metadata standard (id3, XMP, etc.) and which tags do you most recommend for this? please be exact with the tag names (preferably down to the tag ID contained in the standard’s specification). the reason I’m asking is that I’m failing to tag certain files such that the tags appear in Voice.

and, in case a given format/container doesn’t support the best option, what alternatives can you recommend?

**Comment by ian-pop** [comment]
I've been cleaning up my library and have found that MP3 (is that id3?) Artist and Album tags are the drivers of the title and author. 
If you have multiple MP3s in a folder as chapters, make sure that each has the same Artist and Title.
Chapters are the track titles when there are multiple files in a folder.
The cover art came from a single .jpg in the folder or embedded cover art in a stand alone file.
There may be more to it, but that got my library looking how I want.

Edit: I had to uninstall and reinstall the app to have it pick the changes up. 
Moving the book's folder out and back in to my library folder wasn't enough

**Comment by sandreas** [comment]
@chaserene 

I'd recommend the `m4b` format, which is just a renamed `mp4` container file with no video. The most important advantages in my opinion are:

- 1 file per audio book
- Different audio codecs supported, e.g. AAC which produces much smaller files with the same quality

In every case I would set the following metadata fields (kind of must have):

- `Album` - Title of the audio book
- `Artist` - Author(s), comma separated
- `Composer` - Narrator(s), comma separated
- `Genre` - Genre (like `Fantasy`)
- `Description/Comment` - Book-Description (blurb)
- `MovementName` - Series (e.g. Harry Potter), if the book is part of a series
- `MovementIndex` - Numeric Part (e.g. 1), if the book is part of a series
- `Chapters` - The chapter marks, only if it is one file per audio book (mp3 also does support chapters via chapter frame addendum)

Nice to have are these:

- `AlbumSort` - e.g. `Harry Potter 1 - Harry Potter and the philosopher' stone`
- `Part` - e.g. `2.1`, if the part is not an integer value > 0, but contains a `0` or a string
- `ReleaseDate` - e.g. `2024-12-05`
- `Publisher` - e.g. `Remington Steals Audio Books`

Here is a pretty good overview (with some exceptions): 
https://github.com/seanap/Plex-Audiobook-Guide?tab=readme-ov-file#tags-that-are-being-set

And here is a Mapping for different formats (e.g. m4b=mp4 or mp3):
https://docs.mp3tag.de/mapping/
